packages/now-cli/test/dev/integration.js

```diff
@@ -511,13 +508,12 @@ if (satisfies(process.version, '>= 6.9.0 <7.0.0 || >= 8.9.0')) {
   test(
     '[now dev] 06-gridsome',
     testFixtureStdio('06-gridsome', async (t, port) => {
-      const result = fetch(`http://localhost:${port}`);
-      const response = await result;
+      const response = await fetch(`http://localhost:${port}`);
 
       validateResponseHeaders(t, response);
 
       const body = await response.text();
-      t.regex(body, /Hello, world!/gm);
+      t.regex(body, /<div id="app"><\/div>/gm);
```

Comment:
We are actually using the `develop` command now because of the new detectors, so we'll just get the template here